hotswap-agent-core/src/main/java/org/hotswap/agent/config/PluginConfiguration.java

```diff
@@ -226,6 +243,16 @@ private void initExcludedClassLoaderPatterns() {
     }
 
     private boolean checkExcludedClassLoaderPatterns() {
+        if (PluginManager.getInstance().getHotswapTransformer().getIncludedClassLoaderPatterns() != null) {
```

Comment:
why put included-classloader-pattern check logic in 'checkExcludedClassLoaderPattern' method?